File: server/src/room_objects/Coordinates.js
"use strict";

var Coordinates = function(x, y) {
    this.x = x;
    this.y = y;
};

Coordinates.prototype = {
    getX: function() {
        return this.x;
    },

    getY: function() {
    	return this.y;
    },

    getXY: function() {
    	return [this.x, this.y];
    }
};

module.exports = Coordinates;

File: client/ios/WhiteboardiOS/WhiteboardiOS/Coordinates.m
//
//  Coordinates.m
//  WhiteboardiOS
//
//  Created by Chris Funk on 2015-03-13.
//  Copyright (c) 2015 Patrick Murphy. All rights reserved.
//

#import "Coordinates.h"

@implementation Coordinates
int x;
int y;

void setCoordinates( int nx, int ny ) {
    x = nx;
    y = ny;
}

int getX() {
    return x;
}

int getY() {
    return y;
}
@end
